api/v1/mongodbcommunity_types_test.go

```diff
@@ -28,6 +30,61 @@ func TestMongoDB_MongoURI(t *testing.T) {
 	assert.Equal(t, mdb.MongoURI("my.cluster"), "mongodb://my-rs-0.my-rs-svc.my-namespace.svc.my.cluster:40333,my-rs-1.my-rs-svc.my-namespace.svc.my.cluster:40333/?replicaSet=my-rs")
 }
 
+func TestMongoDB_MongoURI_With_Options(t *testing.T) {
+	mdb := newReplicaSet(2, "my-rs", "my-namespace")
```

Comment:
Might be nicer with parametric test, such as this one: https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/test/e2e/replica_set_arbiter/replica_set_arbiter_test.go#L26

Reply:
Done! Although, for these short unit tests, it feels a bit overkill.